javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,61 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+function renderProducts(products) {
+
+    products.forEach(products => {
+        let myUl = document.getElementById("myUl");
```

Comment:
Instead of getting this for every product, get it once and reuse it.
```suggestion
    let myUl = document.getElementById("myUl");
    products.forEach(products => {
```

Reply:
So I should take let myUl outside of a function?

Reply:
Cool, yes, got it Cameron, thanks a lot.